stumpy/stump.py

```diff
@@ -53,6 +65,8 @@ def _get_first_stump_profile(start, T_A, T_B, m, excl_zone, M_T, Σ_T, ignore_tr
     I : int64
         Matrix profile index for the window with index equal to `start`
     """
+    # TODO: return value of I is not simply an int64, looks like it's a tuple.
+    # it's either: (np.ndarray, int, int) or (np.ndarray, np.ndarray, np.ndarray)
```

Comment:
You are right. The docstring is out of date. The return value of `I` should be a tuple of 3x `int64` (e.g., (3, 19, 20) or (10, -1, -1) ). The docstring should probably say something like:

```
I : Tuple[int64, int64, int64]
        Matrix profile index, left matrix profile index, and right matrix profile index for the window with index equal to `start`. The left and right matrix profile indices are automatically set to `-1` for self-joins (i.e., when `ignore_trivial` is set to `True`.
```

@cydave Would you mind updating the docstring (it's not obvious to me how the tuple should be represented there) and the type hint?

Reply:
This looks to me like:
```
Tuple[np.int64, np.int64, np.int64] (if you require a numpy.int64, if you don't care too much about the specific type you could use Tuple[int, int, int]  :)

Try to find the actual type of the variables you're using and use those as the types you expect them to be in.

Reply:
Although on second thought (after searching for a bit) the type might not need to be prefixed with `np.`. So the one you came up with appears to be the correct way to go about it.

Reply:
It’s actually just a single element so I think int is perfectly fine

Reply:
@seanlaw  I think the change at https://github.com/TDAmeritrade/stumpy/pull/93/commits/c37e1d7877e3e78fc6c47a88606b159c1e1fc606  should do it.